edge-modules/metrics-collector/src/README.md

```diff
@@ -69,6 +69,8 @@ Optional config items:
     - Configurable azure domain which is used to construct the log analytics upload address.
     - ex: `azure.com.cn`
     - Defaults to `azure.com`
+- `TransportType`
+    - The transport used to connect to IoT Hub when `UploadTarget` is `IotMessage`. Valid values are `Amqp_Tcp_Only`, `Amqp_WebSocket_Only`, `Mqtt_Tcp_Only`, and `Mqtt_WebSocket_Only`. `Amqp` and `Mqtt` are also valid, and are aliases for `Amqp_Tcp_Only` and `Mqtt_Tcp_Only`, respectively. Default is `Amqp_Tcp_Only`.
```

Comment:
Good callout!

Reply:
adding @micahl to help update [official docs](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-collect-and-transport-metrics?view=iotedge-1.4&tabs=iothub#metrics-collector-configuration).

Reply:
Will need to update default metrics collector image in marketplace from 1.0 to 1.1. 
@jlian may be able to help with that.

Reply:
I'm not sure if I have access to make that update. I checked partner center and am not seeing the metrics collector in my list.

Reply:
How did we publish it before?